site-manage/site-manage-group-section-role-helper/tool/src/java/org/sakaiproject/site/tool/helper/managegroupsectionrole/rsf/GroupListProducer.java

```diff
@@ -126,17 +133,29 @@ public void fillComponents(UIContainer tofill, ViewParameters viewparams,
 				String groupId = group.getId();
 				UIBranchContainer grouprow = UIBranchContainer.make(deleteForm, "group-row:", group.getId());
 				
-				UIOutput.make(grouprow, "group-title-label", group.getTitle());
+				String groupTitle = Validator.escapeHtml(group.getTitle());
```

Comment:
Just curious: what was the reasoning behind escaping the groupTitle ?

This seemed to have introduced the issue that group names containing certain special characters (like 'ä', 'ö', 'ü') are displayed with those characters escaped ('&auml','&ouml', '&uuml').
This just happens with the groupTitle and not with the name of a joinable set containing the same characters.

Can this escapeHtml method-call be safely removed without side effects?